pkg/operator/sync.go

```diff
@@ -36,7 +35,7 @@ func (optr *Operator) syncAll(rconfig render.OperatorConfig) error {
 	})
 }
 
-func (optr *Operator) syncCustomResourceDefinitions(config render.OperatorConfig) error {
+func (optr *Operator) syncCustomResourceDefinitions(config OperatorConfig) error {
 	crds := []string{
 		"/machine.crd.yaml",
```

Comment:
Is there a reason we prepend `/` and not just add those in via `path.Join()` when needed. It seems overloaded here.